root/report/recordings_possibly_assigned_to_the_wrong_artist.tt

```diff
@@ -0,0 +1,17 @@
+[%- WRAPPER 'layout.tt' title=l('Recordings possibly assigned to the wrong artist(s)') full_width=1 -%]
```

Comment:
Although the title is _OK_ for Rochus, I still think it is not specific enough. There are many reasons for recordings to be _possibly assigned to the wrong artist_. The previous ticket title was more specific. Was it unclear or unreadable? Input from a native English speaker such as @mwiencek would be welcome.

Reply:
There are many reasons, you're right, which is precisely the reason I feel we should use this title. In the description I clarified exactly what it means to be 'possibly wrong', but we could change the way this works and add more conditions to evaluate it better and the title would still hold. However, a precise title also makes sense. @mwiencek is the tie breaker here then.

Reply:
I suppose I prefer the precise title ("Recordings with the same name by different artists with the same name").

And actually, I'm not sure it's a good idea to make this report any more complicated (e.g. combining the track AC idea into here) because it would make the report harder to step through for volunteers. They'd have to figure out which of the two cases the problem is each time and switch workflows while fixing them. Having a separate report just makes much more sense to me.

Reply:
Done.